frontend/src/metabase/ui/components/inputs/Combobox/Combobox.config.ts

```diff
@@ -0,0 +1,9 @@
+import { Combobox, type MantineThemeOverride } from "@mantine/core";
+
+export const comboboxOverrides: MantineThemeOverride["components"] = {
```

Comment:
The only reason I'm adding `Combobox` is to use `Combobox.Chevron` which is used by default in `Select`.
`Combobox.Chevron` is needed to customize `CurrencyPicker`.